internal/plugins/helm/v1/scaffolds/templates/manager/config.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Modifications copyright 2020 The Operator-SDK Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &Config{}
+
+// Config scaffolds yaml config for the manager.
+type Config struct {
+	file.TemplateMixin
+
+	// Image is controller manager image name
+	Image string
+
+	// OperatorName will be used to create the pods
+	OperatorName string
+}
+
+// SetTemplateDefaults implements input.Template
+func (f *Config) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = filepath.Join("config", "manager", "manager.yaml")
+	}
+
+	f.TemplateBody = configTemplate
+
+	if f.OperatorName == "" {
+		dir, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("error to get the current path: %v", err)
+		}
+		f.OperatorName = filepath.Base(dir)
+	}
+	return nil
```

Comment:
@joelanford see that it is == you suggested now. 
However, as a follow up I think we should change it to have the OperatorName in the PROJECT file instead and then here just get from the config/project file which would make easier how it would work for SDK the subcommands.